sys/sched_round_robin/sched_round_robin.c

```diff
@@ -19,7 +19,11 @@
 
 #include "sched.h"
 #include "thread.h"
-#include "xtimer.h"
+#if MODULE_ZTIMER_USEC
+    #include "ztimer/xtimer_compat.h"
+#else
+    #include "xtimer.h"
+#endif
```

Comment:
This shouldn't be needed. The idea is that you just add `USEMODULE += ztimer_xtimer_compat` and every call to xtimer is replaced by a corresponding call in `ztimer`.

Reply:
the idea of this include is: "i hear by declare this works with ztimer_xtimer_compat even if somone else wants to have xtimer"

Reply:
and since xtimer_compat.h is an ztimer header it makes the code ztimer compatible without me needing to have a huge number of if `ztimer {do ztimer stuff) else {do xtimer stuff)`